@navikt/core/css/darkside/form/file-upload.darkside.css

```diff
@@ -142,52 +135,55 @@
 }
 
 .navds-dropzone--dragging .navds-dropzone__area-release > .navds-dropzone__area-release__text {
-  transition: visibility var(--__ac-dropzone-animation-length-long) var(--__ac-dropzone-animation-over-under);
+  transition: visibility var(--__axc-dropzone-animation-length-long) var(--__axc-dropzone-animation-over-under);
   visibility: visible;
 }
 
 .navds-dropzone--dragging .navds-dropzone__area-release {
-  color: var(--a-text-action-selected);
+  color: var(--ax-text-accent-strong);
   top: 50%;
   transform: translateY(-50%);
 }
 
 .navds-dropzone__area-disabled {
-  color: var(--a-text-subtle);
+  color: var(--ax-text-subtle);
   display: flex;
   align-items: center;
   flex-direction: column;
-  gap: var(--a-spacing-2);
+  gap: var(--ax-spacing-2);
 }
 
-/**
- * FileUpload.Item
- */
-
+/* ----------------------------- FileUpload Item ---------------------------- */
 .navds-file-item__inner {
```

Comment:
Did we want to match the border color with the outline color on focus? 
![image](https://github.com/user-attachments/assets/24d22fcf-6f2e-453a-87c9-87a49555bc56)
![image](https://github.com/user-attachments/assets/07b82b0a-3f7c-475a-a57d-e72bfe15e1bb)